test/test_linalg.py

```diff
@@ -1445,6 +1447,46 @@ def run_test_case(input, ord, dim, keepdim, norm_dtype):
                             keepdim,
                             norm_dtype)
 
+    def test_vector_norm_decom_unbacked_checks(self):
+        from torch._refs.linalg import _check_vector_norm_args
+
+        class Mod(torch.nn.Module):
+            def __init__(self, ord, dim, add_assert):
+                super().__init__()
+                self.ord=ord
+                self.dim = dim
+
+            def forward(self, a):
+                x = a.item()
+
+                if add_assert:
+                    torch._check(x>100)
+                tensor_unbacked_size = torch.ones((x))
+                _check_vector_norm_args(tensor_unbacked_size, -1, 1)
+                return tensor_unbacked_size
+
+        # When expect_runtime_check we expect a runtime check to inserted.
+        def test(ord: Union[float, int], dim: Optional[DimsType], expect_runtime_check:bool)->None:
+            m = Mod(ord, dim)
+            exported_program: torch.export.ExportedProgram = torch.export.export(
+                m , args= tuple(torch.tensor([1])))
+            print(exported_program.graph_module.code.strip())     
+
+        # # Those are situation where we should throw error if input.size()==0
```

Comment:
Why do we have these commented out ? if its for something that should work, but doesnt, you can add a test with expected failure.

Reply:
ah this is draft i am adding unit tests for got to mark the PR draft